src/makeRobot.js

```diff
@@ -39,6 +39,63 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
```

Comment:
if variables and keys have the same names, you can write them like this

```suggestion
  const robot = {
    name,
    wheels,
    version,
```